src/app/app.module.ts.orig

```diff
@@ -10,12 +10,8 @@ import { AdminDashboardComponent } from './admin/admin-dashboard/admin-dashboard
 import { UserObservationsComponent } from './user/user-observations/user-observations.component';
 import { SharedModule } from './shared/shared.module';
 import { AccountModule } from './account/account.module';
-<<<<<<< HEAD
```

Comment:
This file needs to be removed - not something you want to commit to a central repository.